framer/Material-UI.framerfx/code/Icon.tsx

```diff
@@ -1,43 +1,40 @@
 import * as React from 'react';
-import { pascal } from 'naming-style';
 import { addPropertyControls, ControlType } from 'framer';
-// tslint:disable-next-line: ban-ts-ignore
-// @ts-ignore
 import * as Icons from '@material-ui/icons';
+import { SvgIconProps } from '@material-ui/core/SvgIcon';
+import { pascalCase } from './utils';
 
-interface Props {
-  color?: 'inherit' | 'primary' | 'secondary' | 'action' | 'error' | 'disabled';
-  icon?: string;
-  theme?: 'Filled' | 'Outlined' | 'Rounded' | 'TwoTone' | 'Sharp';
-  width?: number;
-  height?: number;
+interface Props extends SvgIconProps {
+  color: 'action' | 'disabled' | 'error' | 'inherit' | 'primary' | 'secondary';
+  icon: string;
+  theme: 'Filled' | 'Outlined' | 'Rounded' | 'TwoTone' | 'Sharp';
+  width: number | string;
+  height: number;
 }
 
-const defaultProps: Props = {
-  color: 'inherit',
-  icon: 'add',
-  theme: 'Filled',
-  width: 24,
-  height: 24,
-};
-
-export const Icon: React.SFC<Props> = (props: Props) => {
+export function Icon(props: Props): JSX.Element | null {
   const { height, icon: iconProp, theme, width, ...other } = props;
-  const iconName = `${iconProp && pascal(iconProp)}${theme === 'Filled' ? '' : theme}`;
-  // tslint:disable-next-line: ban-ts-ignore
-  // @ts-ignore
-  const Icon = Object.keys(Icons).includes(iconName) ? Icons[iconName] : undefined;
+  const iconName = `${iconProp && pascalCase(iconProp)}${
+    theme === 'Filled' ? '' : theme
+  }` as keyof typeof Icons;
+  const Icon = Object.keys(Icons).indexOf(iconName) !== -1 ? Icons[iconName] : undefined;
```

Comment:
Why?

Reply:
`includes` was introduced in ES7 (es2016). The published tsconfig indicates ES6 (es2015). Since it's unclear what the actual TS environment in framer is I'd rather downgrade it. Especially because `indexOf` vs `includes` has little cost to convert.